test/check-application

```diff
@@ -60,6 +60,15 @@ class TestApplication(testlib.MachineCase):
             findmnt --list -otarget | grep /var/lib/containers/. | xargs -r umount
             """)
 
+        # copy images for user podman tests; podman insists on user session
+        m.execute("""
+            loginctl enable-linger $(id -u admin)
+            for img in quay.io/libpod/busybox quay.io/libpod/alpine quay.io/cockpit/registry:2; do
```

Comment:
It's a bit awkward to have to repeat the container list here. Could we not just restore the tarballs into both system and user, or iterate over `podman images -n` or so?